--- a/node_modules/handlebars/lib/index.js
+++ b/node_modules/handlebars/lib/index.js
@@ -1,6 +1,3 @@
-var fs = require('fs');
-
 module.exports = require('./handlebars/base');
 
 // Every time this file is required, we need to reset the state of the parsers
@@ -8,22 +5,3 @@
   // Reset the parsers
   require('./handlebars/compiler/parser');
 });
-
-// We're not in the browser, so we need to patch require to be able to load templates
-// from the filesystem.
-if (
-  typeof window === 'undefined' &&
-  typeof process !== 'undefined' &&
-  process.title !== 'browser'
-) {
-  var engine = require('./handlebars/compiler/javascript-compiler');
-
-  // We have to support multiple extensions so we pull them from the file system
-  var extensions = ['hbs', 'handlebars'];
-
-  for (var i = 0; i < extensions.length; i++) {
-    require.extensions['.' + extensions[i]] = function (module, filename) {
-      var template = fs.readFileSync(filename, 'utf8');
-      module.exports = engine.compile(template);
-    };
-  }
-}